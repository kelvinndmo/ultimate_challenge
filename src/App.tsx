import React, { FC } from 'react';
import { createMuiTheme, ThemeProvider } from '@material-ui/core';
import './App.css';
import WidgetBuilder from './views';

const theme = createMuiTheme({
  palette: {
    primary: {
      main: '#1098f7',
    },
    secondary: {
      main: '#000103',
    },
  },
});

const App: FC = () => {
  return (
    <ThemeProvider theme={theme}>
      <div
        style={{
          width: '100vw',
          height: '100vh',
          paddingTop: '30px',
        }}
      >
        <WidgetBuilder />
      </div>
    </ThemeProvider>
  );
};

export default App;
