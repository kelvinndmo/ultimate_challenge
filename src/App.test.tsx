import React from 'react';
import { shallow } from 'enzyme';
import App from './App';

describe('App Component', () => {
  const AppComponent = shallow(<App />);
  it('should render app component', () => {
    expect(AppComponent).toMatchSnapshot();
  });
});
