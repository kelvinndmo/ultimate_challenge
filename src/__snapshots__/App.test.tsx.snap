// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component should render app component 1`] = `ShallowWrapper {}`;
